stdlib/public/SDK/CoreImage/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+cmake_minimum_required(VERSION 3.4.3)
+include("../../../../cmake/modules/StandaloneOverlay.cmake")
```

Comment:
Ah, I see what you did. :-/ I don't like this setup because it seems too easy for us to add something to the main CMakeLists.txt file and forget in in the overlay config.